Workshops.KernelAi.ConsoleApp/Modules/SemanticKernel/13_ExternalTemplates.cs

```diff
@@ -0,0 +1,155 @@
+namespace Workshops.KernelAi.ConsoleApp.Modules.SemanticKernel;
+
+public class ExternalTemplates(IAnsiConsole console, WorkshopSettings settings) : IExample
+{
+    public string Name => "Semantic Kernel External Templates";
+
+    public WorkshopModule Module => WorkshopModule.SemanticKernel;
+
+    public async Task RunAsync()
+    {
+        ModelSettings chatSettings = settings.Chat;
+        console.WriteModelInfo(chatSettings);
+
+        Kernel kernel = Kernel.CreateBuilder()
+            .AddWorkshopChatCompletion(chatSettings)
+            .Build();
+
+        console.MarkupLine("[cyan]Semantic Kernel External Templates Demo[/]");
+        console.WriteLine();
+
+        // Load templates from files
+        string templatePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Content", "Templates");
+        
+        if (!Directory.Exists(templatePath))
+        {
+            console.MarkupLine("[red]Error: Templates directory not found![/]");
+            console.MarkupLine($"[red]Expected path: {templatePath}[/]");
+            return;
+        }
+
+        // Load all template files
+        var templates = new Dictionary<string, KernelFunction>();
+        
+        foreach (string templateFile in Directory.GetFiles(templatePath, "*.txt"))
+        {
+            string templateName = Path.GetFileNameWithoutExtension(templateFile);
+            string templateContent = await File.ReadAllTextAsync(templateFile);
+            
+            templates[templateName] = kernel.CreateFunctionFromPrompt(
```

Comment:
This generates a KernelException: A function named argument must contain a name and value separated by a '=' character.